### PR TITLE
NestedFolders: Style tweaks

### PR DESCRIPTION
**Pull request by joshhunt**
- Vertically align folder expand/collapse Align arrows
- Change “Empty folder” message to secondary text color, keep the italics, change text to “No items”

image

- Use Stack and Text components in Move and Delete modals (increase font size of counts to body)

| Move Before | Move After |
| ---------------|-------------|
| image | image |

| Delete Before | Delete After |
| ---------------|-------------|
| image | image |

Part of https://github.com/grafana/grafana/issues/65604

### CODE REVIEW

File: public/app/features/browse-dashboards/components/BrowseActions/DescendantCount.tsx [hunk 1]
```diff
@@ -14,24 +13,16 @@ export interface Props {
...
-    <div className={styles.breakdown}>
-      <>
-        {data && buildBreakdownString(data.folder, data.dashboard, data.libraryPanel, data.alertRule)}
-        {(isFetching || isLoading) && <Spinner size={12} />}
-        {error && <Alert severity="error" title="Unable to retrieve descendant information" />}
-      </>
-    </div>
+    <>
+      {data && (
+        <P color="secondary">{buildBreakdownString(data.folder, data.dashboard, data.libraryPanel, data.alertRule)}</P>
```
Comment (ashharrison90) [inline public/app/features/browse-dashboards/components/BrowseActions/DescendantCount.tsx:21]: the different font size here causes some content shift when combined with the spinner

could just up the spinner size to 14, or we could maybe do something cleverer with getting the value from the theme? 🤔

**Review by ashharrison90 — APPROVED** [review]
minor nit with the font size change causing some content shift in the `DescendantCount` component

otherwise lgtm 👍

File: public/app/features/browse-dashboards/components/BrowseActions/DescendantCount.tsx (hunk 1 above)
Comment (joshhunt) [reply public/app/features/browse-dashboards/components/BrowseActions/DescendantCount.tsx:21]: great catch!

**Review by amy-super — APPROVED** [review]
I'm so happy those arrows are aligned now :) 
Also I think the font size increase looks great! 👍

**Pull request merged**